strawberry/types/fields/resolver.py

```diff
@@ -39,12 +39,16 @@ def __call__(self, *args, **kwargs) -> T:
         return self.wrapped_func(*args, **kwargs)
 
     @cached_property
-    def arguments(self) -> List[StrawberryArgument]:
+    def function_parameters(self) -> Mapping[str, inspect.Parameter]:
+        return inspect.signature(self.wrapped_func).parameters
+
+    @cached_property
+    def arguments(self) -> Mapping[str, Type]:
```

Comment:
Good point, will change.

Reply:
Also what should the second argument here be? Is `Type` correct or should it just be `object`?

Reply:
Oh I meant to comment on it. I think it should be `object` for now. I think in the long term, the arguments' typing can get fancy with Generics, though.

Also, when you do use it, I think the best practice is to use `type` instead of `Type` if you're not using the argument to `Type`

Reply:
Oh wait, I think I was looking at this wrong. I think this should be `Dict[str, StrawberryType]`?